version.txt

```diff
@@ -1 +1 @@
-0.15.2
+0.16.0
```

Comment:
version.txt is updated in a release-pr (automatically) as part of changelog generating etc., so this file should not be updated here (nor will it have any effect).

Reply:
Good to know. I can roll it back